bucket/unciv.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "4.2.12",
+    "homepage": "https://github.com/yairm210/UnCiv",
+    "description": "Open source, mod-friendly Android + Desktop remake of Civ V, made with LibGDX",
+    "license": "MPL-2.0",
+    "suggest": {
+        "JRE": "java/temurin-jre"
+    },

```

Comment:
Out of curiosity, why this JRE and not another such as `java/adoptopenjdk-hotspot-jre`? See https://github.com/Calinou/scoop-games/blob/ddf55bdf2f134913f504eba70a24f6c51dadcf59/bucket/nuncabola.json#L6-L8 for another manifest which specifies a JRE.